core/src/test/scala/unit/kafka/server/ReplicaAlterLogDirsThreadTest.scala

```diff
@@ -243,9 +245,9 @@ class ReplicaAlterLogDirsThreadTest {
       responseCallback = callbackCaptor.capture(),
       isolationLevel = ArgumentMatchers.eq(IsolationLevel.READ_UNCOMMITTED),
       clientMetadata = ArgumentMatchers.eq(None)
-    )).thenAnswer(_ => {
-      callbackCaptor.getValue.apply(Seq((topicPartition, responseData)))
-    })
+    )) thenAnswer new Answer[Unit] {
```

Comment:
Should we keep the` .` before `thenAnswer` and use `[]`? The existing convention is the following.

```
      when(...)
      .thenAnswer(new Answer[Unit] {
          override def answer(invocation: InvocationOnMock): Unit = {
             ...
          }
        })
```